mon-pix/package-lock.json

```
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "mon-pix",
-      "version": "3.190.0",
+      "version": "3.192.0",
```

Restaurer la version d'origine :pray: 
On va gérer ce problème de release dans une PR séparée
```suggestion
      "version": "3.190.0",
```